src/rust/engine/src/nodes.rs

```diff
@@ -301,15 +301,18 @@ impl ExecuteProcess {
   ) -> Result<InputDigests, StoreError> {
     let input_digests_fut: Result<_, String> = Python::with_gil(|py| {
       let value = (**value).as_ref(py);
-      let input_files = lift_directory_digest(externs::getattr(value, "input_digest").unwrap())
-        .map_err(|err| format!("Error parsing input_digest {err}"))?;
+      let input_files = lift_directory_digest(
+        externs::getattr(value, "input_digest")
+          .map_err(|err| format!("Error retrieving `input_digest`: {err}"))?,
+      )
+      .map_err(|err| format!("Error parsing input_digest {err}"))?;
       let immutable_inputs =
         externs::getattr_from_str_frozendict::<&PyAny>(value, "immutable_input_digests")
           .into_iter()
           .map(|(path, digest)| Ok((RelativePath::new(path)?, lift_directory_digest(digest)?)))
           .collect::<Result<BTreeMap<_, _>, String>>()?;
       let use_nailgun = externs::getattr::<Vec<String>>(value, "use_nailgun")
-        .unwrap()
+        .map_err(|err| format!("Failed to get value `use_nailgun`: {err}"))?
```

Comment:
For consistency with the other error messages:
```suggestion
        .map_err(|err| format!("Failed to get field `use_nailgun`: {err}"))?
```